frontend/src/scenes/billing/Billing.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from 'react'
+import { useValues, useActions } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { billingLogic } from './billingLogic'
+import { Card, Progress, Row, Col, Button, Popconfirm, Spin } from 'antd'
+import defaultImg from './../../../public/plan-default.svg'
+import { PlanInterface } from '~/types'
+
+function Plan({ plan, onUpgrade }: { plan: PlanInterface; onUpgrade: (plan: PlanInterface) => void }): JSX.Element {
+    return (
+        <Card>
+            <img src={plan.image_url || defaultImg} alt="" height={100} width={100} />
+            <h3 style={{ fontSize: 22 }}>{plan.name}</h3>
+            <div>
+                <Popconfirm
+                    title={`Sign up for the ${plan.name} now? You will need a bank card.`}
+                    onConfirm={() => onUpgrade(plan)}
+                    okText="Yes"
+                    cancelText="No"
+                >
+                    <Button data-attr="btn-upgrade-now" data-plan={plan.key}>
+                        Upgrade now
+                    </Button>
+                </Popconfirm>
+            </div>
+        </Card>
+    )
+}
+
+export function Billing(): JSX.Element {
+    const { plans, billingSubscription, billingSubscriptionLoading, percentage, strokeColor } = useValues(billingLogic)
+    const { loadPlans, subscribe } = useActions(billingLogic)
+    const { user } = useValues(userLogic)
+
+    const handleBillingSubscribe = (plan: PlanInterface): void => {
+        subscribe(plan.key)
+    }
+
+    useEffect(() => {
+        if (!user?.billing?.plan || user?.billing?.should_setup_billing) loadPlans()
+    }, [user])
```

Comment:
This could go in the billingLogic in an `afterMount` event. All other data fetching for the billing info is there, so it just makes sense for this to be together